.github/automation/commit-msg-check.py

```diff
@@ -36,7 +37,7 @@ def __scopeCheck(msg: str):
     scopesArray = firstLine.split(":")[:-1]
 
     for scopes in scopesArray:
-        numWords = len(scopes.split())
+        numWords = len(re.split(',| ', scopes))
```

Comment:
What does this suppose to do?

Reply:
Let's drop this commit. The issue with checker is addressed in #2286.

Reply:
The new commit_msg_checker makes spaces between scopes mandatory. Is this something we want to enforce?

Reply:
Looking at commits history space after `,` and `:` in scope is the common practice. All deviations are a part of 3% mentioned in #2286. So enforcing space just for the sake of consistency feels right.